src/api/admin/admin.ctrl.js

```diff
@@ -1,33 +1,138 @@
-const { Users } = require('../../models');
+const crypto = require('crypto');
+const { users } = require('../../models');
+const { boards } = require('../../models');
 
-const getAdmin = (req, res) => {
-    const { user_id } = req.params;
-    res.json({
-        user_id,
+const isAdmin = async (email) => {
+    const level = 999;
+    const admin = await users.findOne({
+        where: { email, level },
     });
+
+    return admin;
 };
 
-const postAdmin = async (req, res, next) => {
-    const user_id = Number(req.body.user_id);
+const getUser = async (req, res, next) => {
+    try {
+        const checkAdmin = await isAdmin(req.user.emails[0].value);
+        if (!checkAdmin) throw new Error('NOT_ADMIN');
+
+        // TODO: 관리자 제외? || 탈퇴자 제외?
+        const userList = await users.findAll();
+        res.json({ userList });
+    } catch (err) {
+        next(err);
+    }
+};
 
+const postUser = async (req, res, next) => {
     try {
-        if (user_id !== 10) throw new Error('WRONG_USER_ID');
+        const checkAdmin = await isAdmin(req.user.emails[0].value);
+        if (!checkAdmin) throw new Error('NOT_ADMIN');
 
-        const users = await Users.find({
-            where: {
-                user_id,
-            },
+        // TODO:
+        // userName 필드 validation
+        // email 필드 validation
+        // studentId 필드 validation
+
+        if (!req.body.userName) throw new Error('INVALID_PARAMETERS');
+        if (!req.body.email) throw new Error('INVALID_PARAMETERS');
+        if (!req.body.studentId) throw new Error('INVALID_PARAMETERS');
+
+        await users.create({
+            userName: req.body.userName,
+            email: req.body.email,
+            studentId: req.body.studentId,
```

Comment:
여기는 Joi Validation하면
```suggestion
            userName,
            email,
            sutdentId,
```
이렇게 쓸 수 있을 겁니다